core/zms/src/main/rdl/Policy.rdli

```diff
@@ -181,7 +181,7 @@ resource Assertion DELETE "/domain/{domainName}/policy/{policyName}/version/{ver
     Int64 assertionId; //assertion id
     String auditRef (header="Y-Audit-Ref"); //Audit param required(not empty) if domain auditEnabled is true.
     String resourceOwner (header="Athenz-Resource-Owner"); //Resource owner for the request
-    authorize ("update", "{domainName}:policy.{policyName}");
+    authenticate;
```

Comment:
please update the comment section for the block to include what type of authorization statements will be carried out within the code.